nano/core_test/network.cpp

```diff
@@ -15,43 +15,14 @@ using namespace std::chrono_literals;
 
 TEST (network, tcp_connection)
```

Comment:
This test is now testing the behaviour of simple sockets, which are not used anywhere, rather than testing the node tcp connection handling.

Reply:
seems it tests only that a connection works using the system.io_ctx.